plugins/generic/dublinCoreMeta/DublinCoreMetaPlugin.inc.php

```diff
@@ -0,0 +1,158 @@
+		}
+
+		$i=0;
+		foreach (explode($article->getAuthorString(), ', ') as $author) {
+			$templateMgr->addHeader('dublinCoreAuthor' . $i++, '<meta name="DC.Creator.PersonalName" content="' . htmlspecialchars($author) . '"/>');
+		}
+
+		if (is_a($article, 'PublishedArticle') && ($datePublished = $article->getDatePublished())) {
+			$templateMgr->addHeader('dublinCoreDateCreated', '<meta name="DC.Date.created" scheme="ISO8601" content="' . strftime('%Y-%m-%d', strtotime($datePublished)) . '"/>');
+		}
+		$templateMgr->addHeader('dublinCoreDateSubmitted', '<meta name="DC.Date.dateSubmitted" scheme="ISO8601" content="' . strftime('%Y-%m-%d', strtotime($article->getDateSubmitted())) . '"/>');
+		if ($issue && ($datePublished = $issue->getDatePublished())) {
+			$templateMgr->addHeader('dublinCoreDateIssued', '<meta name="DC.Date.issued" scheme="ISO8601" content="' . strftime('%Y-%m-%d', strtotime($issue->getDatePublished())) . '"/>');
+		}
+		if ($dateModified = $article->getDateStatusModified()) $templateMgr->addHeader('dublinCoreDateModified', '<meta name="DC.Date.modified" scheme="ISO8601" content="' . strftime('%Y-%m-%d', strtotime($dateModified)) . '"/>');
+		$i=0;
+		if ($abstracts = $article->getAbstract(null)) foreach($abstracts as $locale => $abstract) {
+			$templateMgr->addHeader('dublinCoreAbstract' . $i++, '<meta name="DC.Description" xml:lang="' . htmlspecialchars(substr($locale, 0, 2)) . '" content="' . htmlspecialchars(strip_tags($abstract)) . '"/>');
+		}
+
+		$i=0;
+		if (is_a($article, 'PublishedArticle')) foreach($article->getGalleys() as $galley) {
+			if (is_a($galley, 'SupplementaryFile')) continue;
+			$templateMgr->addHeader('dublinCoreFormat' . $i++, '<meta name="DC.Format" scheme="IMT" content="' . htmlspecialchars($galley->getFileType()) . '"/>');
+		}
+
+		$templateMgr->addHeader('dublinCoreIdentifier', '<meta name="DC.Identifier" content="' . htmlspecialchars($article->getBestArticleId($journal)) . '"/>');
+
+		if ($pages = $article->getPages()) {
+			$templateMgr->addHeader('dublinCorePages', '<meta name="DC.Identifier.pageNumber" content="' . htmlspecialchars($pages) . '"/>');
+		}
+
+		foreach($templateMgr->get_template_vars('pubIdPlugins') as $pubIdPlugin) {
+			if ($pubId = $pubIdPlugin->getPubId($article)) {
+				$templateMgr->addHeader('dublinCorePubId' . $pubIdPlugin->getPubIdDisplayType(), '<meta name="DC.Identifier.' . htmlspecialchars($pubIdPlugin->getPubIdDisplayType()) . '" content="' . htmlspecialchars($pubId) . '"/>');
+			}
+		}
+
+		$templateMgr->addHeader('dublinCoreUri', '<meta name="DC.Identifier.URI" content="' . $request->url(null, 'article', 'view', array($article->getBestArticleId($journal))) . '"/>');
+		$templateMgr->addHeader('dublinCoreLanguage', '<meta name="DC.Language" scheme="ISO639-1" content="' . substr($article->getLocale(), 0, 2) . '"/>');
```

Comment:
just a note: differently to google scholar metadata here is the article locale and not language used...